imgui.go

```diff
@@ -658,3 +658,36 @@ func IsItemHovered() bool {
 func IsKeyPressed(key int) bool {
 	return C.iggIsKeyPressed(C.int(key)) != 0
 }
+
+// ProgressBarV creates a progress bar.
+// size (for each axis) < 0.0f: align to end, 0.0f: auto, > 0.0f: specified size
+func ProgressBarV(fraction float32, size Vec2, overlay string) {
+	sizeArg, _ := size.wrapped()
+	overlayArg, overlayFin := wrapString(overlay)
+	defer overlayFin()
+	C.iggProgressBar(C.float(fraction), sizeArg, overlayArg)
+}
+
+// ProgressBar calls ProgressBarV(fraction, Vec2{-1, 0}, "")
+func ProgressBar(fraction float32) {
+	ProgressBarV(fraction, Vec2{-1, 0}, "")
+}
+
+// AlignTextToFramePadding vertically aligns upcoming text baseline to
+// FramePadding.y so that it will align properly to regularly framed
+// items (call if you have text on a line before a framed item)
+func AlignTextToFramePadding() {
+	C.iggAlignTextToFramePadding()
+}
+
+// SetIniFileName sets file name used to load and save
+// windows layout. Call with an empty string to disable INI file.
+// This function can be only called before first call to NewFrame()
+func SetIniFileName(name string) {
```

Comment:
The property is of the IO structure. Please add the function on the IO type as well and move the wrapper functions into the respective files as well.